src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+import chatMessages from './data/messages.json'
+// import ChatEntry from './components/ChatEntry'
+import ChatLog from './components/ChatLog'
+import {useState} from 'react';
```

Comment:
Small nit, since L1 and L6 both import classes/functions from `react`, you can combine them in the same line:
```
import React, { useState } from 'react';
```